plugins/widgetinspector/widgetinspectorserver.cpp

```diff
@@ -195,15 +196,18 @@ void WidgetInspectorServer::widgetSelected(const QItemSelection &selection)
     if (m_selectedWidget
         && (qobject_cast<QDesktopWidget *>(m_selectedWidget)
             || m_selectedWidget->inherits("QDesktopScreenWidget"))) {
-        m_overlayWidget->placeOn(0);
+        m_overlayWidget->placeOn({});
```

Comment:
{} will probably be rejected by MSVC 2010/2012 (which are only in the nightly builds, so you need to trigger a build there manually to verify before integration).